src/manifestoo/main.py

```diff
@@ -71,6 +71,15 @@ def callback(
             "This option is useful in combination with `--select-addons-dir`."
         ),
     ),
+    select_addons_categories: Optional[str] = typer.Option(
+        None,
+        metavar="addon1,addon2,...",
+        help=(
+            "Comma separated list of addons categories to select. "
+            "Only addons belonging to these categories will be searched in "
+            "the addons path."
+        ),
+    ),
```

Comment:
Can you move it after `exclude_core_addons` and rename it `--filter-categories`.
Also update the help text. I think this must not be a selector but a filter, so it should not select in the addons path but filter whatever has been selected - this is more versatile.

This in terms for implementation, this must be done after all selection (similar to select_exclude or exclude_core_addons).